ipa-core/src/report/hybrid.rs

```diff
@@ -239,9 +296,9 @@ mod test {
         }
     }
 
-    fn generate_random_bytes(size: usize) -> Vec<u8> {
+    fn generate_random_bytes() -> [u8; 16] {
```

Comment:
```suggestion
    fn generate_random_bytes() -> [u8; TAG_SIZE] {
```

Reply:
Based on the usage, it seems that you simply want `fn generate_random_tag() -> UniqueTag`

Reply:
oh good call out. I made a simple change here (since it used to be bytes), but your suggestion makes sense!